registry.json

```diff
@@ -1,4 +1,19 @@
 [
+  {
+      "ticker":"ATDT",
+      "poolId":"ae7015d54c634f922f800d9800529c3c5a007148ab9d792ac4ff33ae",
+      "social":{
+         "facebook":"",
+         "twitter":"atdtpool",
+         "website":"https://pool.atdt.io/",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"",
```

Comment:
We require a Telegram handle for you to register. This needs to be for an individual and not a channel or group

Reply:
done!